src/encoded/schemas/experiment_series.json

```diff
@@ -0,0 +1,211 @@
+{
+    "title": "Experiment series",
+    "description": "Schema for submitting metadata for a group of experiments which are applying the same assay on the same type of biosample.",
+    "id": "/profiles/experiment_series.json",
+    "$schema": "http://json-schema.org/draft-04/schema#",
+    "type": "object",
+    "required": [ "related_datasets"],
+    "identifyingProperties": ["uuid", "accession", "aliases"],
+    "additionalProperties": false,
+    "mixinProperties": [
+        { "$ref": "mixins.json#/schema_version" },
+        { "$ref": "mixins.json#/uuid" },
+        { "$ref": "mixins.json#/accession" },
+        { "$ref": "mixins.json#/aliases" },
+        { "$ref": "mixins.json#/attribution" },
+        { "$ref": "mixins.json#/submitted" },
+        { "$ref": "mixins.json#/notes" },
+        { "$ref": "mixins.json#/submitter_comment" },
+        { "$ref": "series.json#/properties" }
```

Comment:
missing dataset.json/#properties?

Reply:
That's on purpose to not have other properties. Should I have them, like `alternate_accessions`,  `dbxrefs`, `internal_tags`, and have `status` defined that way?

Reply:
I agree with @hitz. Since ExperimentSeries is a kind of dataset, I was already surprised to find it didn’t inherit from the dataset schema, even if some properties aren’t used.